### ISSUE TITLE
Keyboard focus not assigned to "Heading style" dropdown, cannot select via keyboard

### ISSUE THREAD

**Issue by jimbo1qaz** [mozilla/notes#1343]
### [Description]:
When I open a note and click on the "Aa↓" heading/body selector, keyboard focus remains in the document instead of the list of styles.

### [Affected versions]:
Desktop 4.2.0

### [Prerequisites]:
Install the extension and create a note. Optionally add some text.

### [Steps to reproduce]:
- Open a note, focus cursor in text field.
- Click "Aa↓" to open the "Heading 1/2/3, Body" dropdown list.
- Press keyboard Up/Down arrows.

### [Expected result]:
Keyboard scrolls through heading and body styles.

### [Actual result]:
Caret scrolls through note text.
